cegs_portal/search/views/v1/reg_effects.py

```diff
@@ -1,11 +1,15 @@
 from django.core.paginator import Paginator
 
-from cegs_portal.search.json_templates.v1.reg_effect import (
+from cegs_portal.search.json_templates.v1.source_reg_effects import (
```

Comment:
I think that `source_reg_effects` should come from `cegs_portal.search.json_templates.v1.source_reg_effects` but that `regulatory_effect` should come from `cegs_portal.search.json_templates.v1.reg_effect`